hbase/versions.py

```diff
@@ -23,6 +25,22 @@
         "phoenix": "5.2.0",
         "hbase_profile": "2.4",
         "hadoop": "3.3.6",
-        "jmx_exporter": "1.0.1",
+        "jmx_exporter": "1.0.1",  # update the stackable/jmx/config<version> folder too
+        "opa_authorizer": "",  # only for HBase 2.6.0
+        "delete_caches": "true",
+    },
+    {
+        "product": "2.6.0",
+        "hbase_thirdparty": "4.1.7",
+        "hbase_operator_tools": "1.3.0-SNAPSHOT",
```

Comment:
We need this to be versioned _somehow_.
For example by referencing a git commit hash

Same for phoenix